### ISSUE TITLE
Add timeout / catch exception when we cannot obtain the issuer profile.

### ISSUE THREAD

**Issue by raiseandfall** [blockchain-certificates/cert-verifier-js#22]
With a specific error message "Could not obtain issuer profile".

**Comment by kimdhamilton** [comment]
more broadly, we should timeout and catch when any lookup fails. The error message should reflect that

**Comment by lemoustachiste** [comment]
@raiseandfall would that still be an issue? With the request method if there is an error it is caught and an error is thrown. However indeed we don't handle any timeouts.

**Comment by raiseandfall** [comment]
@lemoustachiste True. Handling global timeouts might have to be a different ticket, no?
